src/PostgREST/SchemaCache.hs

```diff
@@ -375,32 +375,40 @@ accessibleFuncs = SQL.Statement sql params decodeFuncs
       (snd >$< arrayParam HE.text)
     sql = funcsSqlQuery <> " AND has_function_privilege(p.oid, 'execute')"
 
-funcsSqlQuery :: SqlQuery
-funcsSqlQuery = encodeUtf8 [trimming|
- -- Recursively get the base types of domains
-  WITH
+baseTypesCte :: Text
+baseTypesCte = [trimming|
+  -- Recursively get the base types of domains
   base_types AS (
     WITH RECURSIVE
     recurse AS (
       SELECT
         oid,
         typbasetype,
-        COALESCE(NULLIF(typbasetype, 0), oid) AS base
+        typnamespace AS base_namespace,
+        COALESCE(NULLIF(typbasetype, 0), oid) AS base_type
       FROM pg_type
       UNION
       SELECT
         t.oid,
         b.typbasetype,
-        COALESCE(NULLIF(b.typbasetype, 0), b.oid) AS base
+        b.typnamespace AS base_namespace,
+        COALESCE(NULLIF(b.typbasetype, 0), b.oid) AS base_type
       FROM recurse t
       JOIN pg_type b ON t.typbasetype = b.oid
     )
     SELECT
       oid,
-      base
+      base_namespace,
+      base_type
     FROM recurse
     WHERE typbasetype = 0
   ),
```

Comment:
This comma here would prevent usage in a single CTE, so the comma should be specified where this CTE snippet is applied 
```suggestion
  )
```

Reply:
True, done.